assets/src/edit-story/components/inspector/prepublish/components/autoVideoOptimization.js

```diff
@@ -69,14 +69,14 @@ const AutoVideoOptimization = ({
       </DescriptionText>
       <ToggleGroup>
         <StyledToggle
-          id="automatic-video-optimization-toggle"
+          id="auto-video-optimization-toggle"
           aria-label={__('Enable automatic video optimization', 'web-stories')}
           checked={areVideosAutoOptimized || false}
           onChange={onAutoOptimizeVideoClick}
         />
         <DescriptionText
           forwardedAs="label"
-          htmlFor="automatic-video-optimization-toggle"
+          htmlFor="auto-video-optimization-toggle"
```

Comment:
These ids were the same as the ids in the old checklist. This won't matter once it's removed.